azure-functions-sdk/src/commands/run.rs

```diff
@@ -45,10 +44,16 @@ impl<'a> Run<'a> {
                     .help("The port to forward to the Azure Functions Host for HTTP requests. Default is 8080."),
             )
             .arg(
-                Arg::with_name("image")
-                    .value_name("IMAGE")
-                    .help("The image of the Azure Function application to run. Default is based off the crate name.")
-                    .index(1),
+                Arg::with_name("script_root")
+                    .long("script-root")
+                    .short("r")
+                    .value_name("ROOT")
+                    .help("The directory to use for the Azure Functions application script root. Default is a temporary directory."),
+            )
+            .arg(Arg::with_name("cargo_option")
```

Comment:
nit: maybe should be called `cargo_options`?

Reply:
I'll fix.